packages/react-keytips/src/utilities/isTargetVisible.ts

```diff
@@ -1,10 +1,12 @@
-import type { PositioningProps } from '@fluentui/react-positioning';
+import { isDisabled } from './isDisabled';
 
 export const isTargetVisible = (
-  target?: PositioningProps['target'],
+  target?: HTMLElement,
   win?: Document['defaultView']
 ): boolean => {
   if (!target || !win) return false;
+  if (isDisabled(target)) return false;
+
   const style = win.getComputedStyle(target as HTMLElement);
```

Comment:
Shouldn't the `target` already be of the HTMLElement type?

Reply:
leftover, good catch